### PR TITLE
[MIG] beesdoo_inventory: migrate to 12.0

### PR DESCRIPTION
**Pull request by sdegueldre**
This PR migrates the beesdoo_inventory module to odoo v12.0

### CODE REVIEW

File: beesdoo_inventory/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import models
```
Comment (remytms) [inline beesdoo_inventory/__init__.py]: Missing end of line.

File: beesdoo_inventory/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+    'name': "beesdoo_inventory",
```
Comment (remytms) [inline beesdoo_inventory/__manifest__.py]: Can you change it by: `Beesdoo Inventory`.

File: beesdoo_inventory/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+    'summary': """
+        Modification of inventory data for the needs of beescoop
+        - SOO24 - Bon de livraison""",
```
Comment (remytms) [inline beesdoo_inventory/__manifest__.py]: Can you change this description to a better one ? :) Thanks.

Comment (remytms) [reply beesdoo_inventory/__manifest__.py]: I mean changing summary, not the description that you can leave empty.

File: beesdoo_inventory/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    # any module necessary for this one to work correctly
+    'depends': ['delivery', 'beesdoo_base'],
+
+    # always loaded
+    'data': [
+        'views/stock.xml'    ],
+
+    # only loaded in demonstration mode
+    'demo': [],
+}
```
Comment (remytms) [inline beesdoo_inventory/__manifest__.py]: Missing end of line.

File: beesdoo_inventory/models/__init__.py [hunk 5]
```diff
@@ -0,0 +1 @@
+from . import stock
```
Comment (remytms) [inline beesdoo_inventory/models/__init__.py]: Missing end of line.

File: beesdoo_inventory/views/stock.xml [hunk 6]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (remytms) [inline beesdoo_inventory/views/stock.xml:1]: Can you fix this file by replacing all tabs by 4 spaces. :)

File: beesdoo_inventory/models/stock.py [hunk 7]
```diff
@@ -1,5 +1,4 @@
-# -*- coding: utf-8 -*-
-from openerp import models, fields, api, _
+from odoo import models, fields, api, _
```
Comment (remytms) [inline beesdoo_inventory/models/stock.py]: Can you use `isort` to sort this import ? :)

**Review by remytms — CHANGES_REQUESTED** [review]
I know that many changes I requested is not from you but it's better to clean it know than never. :)

**Comment by sdegueldre** [comment]
Should be okay now @remytms

**Review by remytms — APPROVED** [review]
Thanks for the improvements ! :)

**Comment by remytms** [comment]
@sdegueldre Same as here: https://github.com/beescoop/Obeesdoo/pull/114#issuecomment-575652417

**Pull request merged**